cluster-scope/base/operators.coreos.com/subscriptions/camel-k/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: camel-k
+  namespace: operators
+spec:
+  channel: stable-1.6
+  installPlanApproval: Automatic
+  name: camel-k
+  source: operatorhubio-catalog
```

Comment:
I don't think we have such catalog available here. In operator hub we have 2 options how to install camel-k:

1. comminuty version of the operator named `camel-k` from `community-operators` catalog
2. redhat certified version with name `red-hat-camel-k` from `redhat-operators` catalog

Based on channel selected I think you wanted to use `community-operators` catalog.

You may also consider updating the channel to `stable-1.7`


These are your options (package name and the catalog can be seen in the URL)

Either community operator:
![image](https://user-images.githubusercontent.com/7453394/143219320-1571f040-ccb1-49f0-b0d7-78102d9bbf00.png)

Or the Red Hat version of it:
![image](https://user-images.githubusercontent.com/7453394/143219439-bc682e94-018c-408f-a536-29e2a3e350ad.png)